virttest/qemu_devices/qdevices.py

```diff
@@ -927,8 +927,22 @@ def verify_hotplug(self, out, monitor):
         return False
 
 
-class Dimm(QDevice):
+class Nvdimm(QDevice):
```

Comment:
I didn't remove the Dimm class, just add a new Nvdimm class, which will not affect hotplug/unplug function.  Anyway, the test for hotplug passed.  
(1/1) smp_16.4096m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.providers.hotplug_memory.before.vm_system_reset.hotplug.backend_file.2M_hugepage.policy_default.two: PASS (157.45 s)